lib/rubocop/cop/rspec/minitest_assertions.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Check if using Minitest matchers.
+      #
+      # @example
+      #   # bad
+      #   assert_equal(a, b)
+      #   assert_equal a, b, "must be equal"
+      #   refute_equal(a, b)
+      #
+      #   # good
+      #   expect(a).to eq(b)
+      #   expect(a).to(eq(b), "must be equal")
+      #   expect(a).not_to eq(b)
+      #
+      class MinitestAssertions < Base
+        extend AutoCorrector
+
+        MSG = 'Use `%<prefer>s` to matcher.'
+        RESTRICT_ON_SEND = %i[assert_equal refute_equal].freeze
+
+        # @!method arguments(node)
+        def_node_matcher :arguments, <<-PATTERN
+          (send nil? {:assert_equal :refute_equal} $_ $_ $_?)
+        PATTERN
+
+        def on_send(node)
+          arguments(node) do |exp, act, msg|
+            prefer = replacement(node, exp, act, msg.first)
+            add_offense(node, message: message(prefer)) do |corrector|
+              corrector.replace(node, prefer)
+            end
+          end
+        end
+
+        private
+
+        def replacement(node, exp, act, msg = nil)
```

Comment:
```suggestion
        def replacement(node, expected, actual, failure_message)
```
I've removed the default value as we seem to always pass it